Determine version of libaec from header does not work and causes error
Author: Cintia Willemyns (Vrije Universiteit Brussel)
--- imagecodecs-2024.1.1.orig/imagecodecs/_aec.pyx	2024-05-27 15:12:24.533724000 +0200
+++ imagecodecs-2024.1.1/imagecodecs/_aec.pyx	2024-05-27 15:13:02.238325670 +0200
@@ -76,10 +76,7 @@
 
 def aec_version():
     """Return libaec library version string."""
-    return (
-        f'libaec {AEC_VERSION_MAJOR}.{AEC_VERSION_MINOR}.{AEC_VERSION_PATCH}'
-    )
-
+    return 'libaec 1.0.6'
 
 def aec_check(data):
     """Return whether data is AEC encoded."""
